packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -245,6 +245,39 @@ void main() {
     expect(previousPageCompleted, true);
   });
 
+  testWidgets('PageController with viewportFraction and only one child', (WidgetTester tester) async {
```

Comment:
This is a good test! It will be perfect if we can also test the area 0~150 and 450~600 and assert the tap does no trigger.
Also, we encourage to have another test that with viewfraction = 1 and assert the tap trigger anywhere. It does not have to be complete, just checking some points should be enough(~0 , ~middle, ~right boundary)

Reply:
@chunhtai Thanks for your review! I've just added more expectations for those areas, to be more specific, I am testing:
- 0.5 pixel before left boundary
- left boundary
- 0.5 pixel before right boundary
- right boundary
So now we know where exactly the triggerable area starts and ends.

Also added test for viewFraction=1 - is that what you meant?

Reply:
yes, I saw the update to the tests, looks great!